README.md

```diff
@@ -145,6 +145,15 @@ vars:
     microsoft_ads_<default_source_table_name>_identifier: your_table_name 
 ```
 
+#### Change how ad name is determined
```

Comment:
Can you move this section farther up, like above the `Change the build schema` section?